goon/browserassets/css/browserOutput-dark.css

```diff
@@ -250,6 +250,7 @@ em						{font-style: normal;	font-weight: bold;}
 						{color: #638500;	text-decoration: underline;}
 .motd a, .motd a:link, .motd a:visited, .motd a:active, .motd a:hover
 						{color: #638500;}
+.darkmblue 				{color: #6685f5;}
```

Comment:
Creating this as a rule here will break the light chat. Please re-create this rule in `browserOutput.css`

Reply:
So the same rules needs to be in both files? I'll fix it in a moment